### ISSUE TITLE
Short question about Discriminator Loss

### ISSUE THREAD

**Issue by bizcjh1027** [affinelayer/pix2pix-tensorflow#127]
![image](https://user-images.githubusercontent.com/18678655/42145864-dd201da6-7dfe-11e8-8e78-4fe0432303b7.png)

As you can see above, my discriminator loss fluctuate near the 0.5.

Does that mean training result is quite successful? judging by an optimal probability of discriminator is 1/2(which means discriminator isn't sure about which is fake or real).

Could you give me an advise for it?

**Comment by vythaihn** [comment]
I think optimal probability of discriminator 1/2 should be the output of D, not the loss function. So, the corresponding value of loss should be around 0.69
